SIS_import/sis_set_url.rb

```diff
@@ -350,7 +350,12 @@ def deleteUrlForUnpublishedSections(termId, setSectionPublished)
 	end
 
 	# splite the comma separated SIS IDs, into set
-	sectionArray = sectionString.split(",")
+	@logger.info sectionString
```

Comment:
It might be better just to force sectionString to always be string. So after the nil check just change this to

`
sectionString = sectionString.to_s
sectionArray = sectionString.split(",")
`

This would convert integers to strings and do nothing for strings. The split works fine if it's a string.

Reply:
Sure. I made the change as suggested and the script runs successfully with the change.